packages/atomic/project.json

```diff
@@ -84,9 +91,24 @@
       }
     },
     "dev": {
-      "dependsOn": ["storybook", "stencil:dev", "cem:dev"],
+      "dependsOn": ["storybook", "build:watch", "cem:dev", "web:dev"],
       "executor": "nx:noop"
     },
+    "build:watch": {
+      "executor": "nx:run-commands",
+      "options": {
+        "command": "node ./scripts/watch.mjs",
+        "cwd": "{projectRoot}",
+        "waitUntilTargets": ["^dev"]
+      }
+    },
+    "web:dev": {
+      "executor": "nx:run-commands",
+      "options": {
+        "command": "vite serve dev",
+        "cwd": "{projectRoot}"
+      }
+    },
```

Comment:
A bit unrelated but I would love for this playground app one day to be separated from the package/atomic folder one day. What do you guys think ?

Reply:
Actually I think this would be required as I currently issues if I tarball atomic from this branch and use it in qa-atomic... This sample is biased since it is in the packages/atomic folder and has all of its dependencies.

```
npm run build

> build
> npx vite build

vite v6.0.2 building for production...
✓ 24 modules transformed.
x Build failed in 72ms
error during build:
[vite]: Rollup failed to resolve import "lit" from "/Users/alexprudhomme/dev/qa-atomic/search/node_modules/@coveo/atomic/dist/components/components/search/atomic-text/atomic-text.js".
This is most likely unintended because it can break your application at runtime.
If you do want to externalize this module explicitly add it to
`build.rollupOptions.external`
    at viteWarn (file:///Users/alexprudhomme/dev/qa-atomic/search/node_modules/vite/dist/node/chunks/dep-A4nAWF7x.js:50718:17)
    at onRollupWarning (file:///Users/alexprudhomme/dev/qa-atomic/search/node_modules/vite/dist/node/chunks/dep-A4nAWF7x.js:50750:5)
    at onwarn (file:///Users/alexprudhomme/dev/qa-atomic/search/node_modules/vite/dist/node/chunks/dep-A4nAWF7x.js:50416:7)
    at file:///Users/alexprudhomme/dev/qa-atomic/search/node_modules/rollup/dist/es/shared/node-entry.js:19475:13
    at Object.logger [as onLog] (file:///Users/alexprudhomme/dev/qa-atomic/search/node_modules/rollup/dist/es/shared/node-entry.js:21201:9)
    at ModuleLoader.handleInvalidResolvedId (file:///Users/alexprudhomme/dev/qa-atomic/search/node_modules/rollup/dist/es/shared/node-entry.js:20090:26)
    at file:///Users/alexprudhomme/dev/qa-atomic/search/node_modules/rollup/dist/es/shared/node-entry.js:20048:26
```

Reply:
I am not sure why this happens 🤔